File: Python/0485_max_consecutive_ones/solution.py
def max_consecutive_ones(nums):
    result = str()
    for num in nums:
        if num == 1:
            result += str(num)
        else:
            result += " "
    result = result.split(" ")
    return len(max(result))

nums = [1,1,0,1,1,1]
print(max_consecutive_ones(nums))

File: JavaScript/0485_max_consecutive_ones/solution.js
function max_consecutive_ones(nums) {
    let result = "";
    for (let num of nums) {
        if (num == 1) {
            result += num;
        } else {
            result += " ";
        }
    }
    result = result.split(" ");
    return Math.max(...result).toString().length;
}

let nums = [1,1,0,1,1,1];
console.log(max_consecutive_ones(nums));